### PR TITLE
Create mail.masseo.json

### PR DESCRIPTION
**Pull request by matte-oss**
## Requirements
Unless explicitly specified otherwise by a **maintainer** or in the requirement description, your domain must pass **ALL** the indicated requirements above.

Please note that we reserve the rights not to accept any domain at our own discretion.

- [V] The file is in the `domains` folder and is in the JSON format.
- [V] The file's name is all lowercased and alphanumeric. 
- [V] You have completed your website. 
- [V] The website is reachable.  
- [V] The CNAME record doesn't contain `https://` or `/`.  
- [V] There is sufficient information at the `owner` field.  

## Website Link/Preview

I'm hosting Zoho mail

### CODE REVIEW

**Comment by cupglassDEV** [comment]
@matte-oss Register `masseo.json` first

**Comment by matte-oss** [comment]
> @matte-oss Register `masseo.json` first

Ok give me some time, i'm working on the website

File: domains/mail.masseo.json
Comment (Stef-00012) [inline domains/mail.masseo.json:1]: masseo.is-a.dev doesn't exist so you can't create a subdomain of it

**Review by Stef-DP — CHANGES_REQUESTED** [review]

**Comment by matte-oss** [comment]
#17106

**Review by Stef-DP — APPROVED** [review]

**Comment by Stef-00012** [comment]
william broke something and i'm unable to merge :skull:

**Comment by matte-oss** [comment]
> william broke something and i'm unable to merge :skull:

Lol

**Comment by Stef-00012** [comment]
well, gotta wait for him to fix the branch rules

**Comment by matte-oss** [comment]
Ty!

**Comment by matte-oss** [comment]
Ty!

**Pull request merged**